workflow.md

```diff
@@ -6,7 +6,7 @@ scikit-learn-contrib is a github organization for gathering high-quality scikit-
 * Collaboration: volunteers from around the world can join you in improving the project
 * Quality: benefit from scikit-learn's experience in producing high-quality machine learning software
 * Transfer: most popular estimators can eventually be promoted to scikit-learn
-* URL: http://contrib.scikit-learn.org/project-name
+* URL: http://contrib.scikit-learn.org
```

Comment:
This shouldn't be rendered as a link. Put it in backticks

Reply:
Do you mean like this:
```
* URL: http://contrib.scikit-learn.org
```

Reply:
`[http://contrib.scikit-learn.org](http://contrib.scikit-learn.org)`